manifest.json

```diff
@@ -46,7 +46,9 @@
         "downloads",
         "topSites",
         "clipboardRead",
-        "clipboardWrite"
```

Comment:
Please make this change only for firefox, you can refer to https://github.com/brookhong/Surfingkeys/blob/master/gulpfile.js#L163